src/Components/Modal/Modal.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import Modal, { ModalProps } from "react-native-modal";
+import { isWeb } from "src/Helpers/platform";
+import WebModal from "./WebModal";
+
+const HybridModal = (props: ModalProps) => {
+  if (!props.isVisible) return null;
+
+  if (!isWeb) {
+    return <Modal {...props}>{props.children}</Modal>;
+  }
+
+  return <WebModal {...props} />;
+};
+
+export default React.memo(HybridModal);
```

Comment:
Why do we need moemoization here?

The component is really small and it does not have any logic.

[In this article](https://dmitripavlutin.com/use-react-memo-wisely/) there is a simple rule of thumb: don’t use memoization if you can’t quantify the performance gains.

And this component is extra small and not rerendered often, so there are no perfamance gains at all